editor/src/components/canvas/canvas-utils.ts

```diff
@@ -2292,9 +2293,8 @@ export function moveTemplate(
                     updatedUtopiaComponents,
                   )
 
-                  const insertResult = insertElementAtPath_DEPRECATED(
+                  const insertResult = insertJSXElementChild(
                     workingEditorState.projectContents,
-                    workingEditorState.canvas.openFile?.filename ?? null,
                     childInsertionPath(underlyingNewParentPath),
```

Comment:
I guess this should be updated as well if the parent is a conditional (I suppose the `getInsertionPathWithSlotBehavior` helper could be used here)

Reply:
great idea, done in https://github.com/concrete-utopia/utopia/pull/3633#:~:text=use%20getInsertionPathWithSlotBehavior-,1bc4bf4,-Merge%20state